docs/references/config/config.toml.md

```diff
@@ -0,0 +1,913 @@
+
+The port has to point to the node's P2P port.
+
+Example with a node on a non-routable network:
+- Node has local IP address `10.10.10.10` and uses port `10000` for P2P communication, set in [`p2p.laddr`](#p2pladdr).
+- The network gateway has the public IP `1.2.3.4` and we want to use publicly open port `26656` on the IP address.
+- A redirection has to be set up from `1.2.3.4:26656` to `10.10.10.10:1000`
+- The `external_address` has to be set to `1.2.3.4:26656`.
+
+### p2p.seeds
+Comma-separated list of seed nodes that the node will try to connect.
+```toml
+seeds = ""
+```
+
+| Value type                        | string (comma-separated list)           |
+|:----------------------------------|:----------------------------------------|
+| **Possible values within commas** | nodeID@IP:port (`"abcd@1.2.3.4:26656"`) |
+|                                   | `""`                                    |
+
+Seed nodes will transmit a P2P address book for the node and then disconnect. No blocks or consensus communication takes
+place.
+
+Example:
+```toml
+seeds = "abcd@1.2.3.4:26656,deadbeef@5.6.7.8:10000"
+```
+
+### p2p.persistent_peers
+Comma-separated list of nodes to keep persistent connections to.
+```toml
+persistent_peers = ""
+```
+
+| Value type                        | string (comma-separated list)           |
+|:----------------------------------|:----------------------------------------|
+| **Possible values within commas** | nodeID@IP:port (`"abcd@1.2.3.4:26656"`) |
+|                                   | `""`                                    |
+
+Persistent peers do not count toward [`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) or
+[`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers).
+
+Example:
+```toml
+persistent_peers = "fedcba@11.22.33.44:26656,beefdead@55.66.77.88:20000"
+```
+
+### p2p.addr_book_file
+Path to the address book file.
+```toml
+addr_book_file = "config/addrbook.json"
+```
+
+| Value type          | string                                                      |
+|:--------------------|:------------------------------------------------------------|
+| **Possible values** | relative directory path, appended to `$CMTHOME` |
+|                     | absolute directory path                         |
+
+The default relative path translates to `$CMTHOME/config/addrbook.json`. In case `$CMTHOME` is unset, it defaults to
+`$HOME/.cometbft/config/addrbook.json`.
+
+### p2p.addr_book_strict
+Strict address routability rules disallow non-routable IP addresses in the address book. When `false`, private network
+IP addresses are enabled to be stored in the address book.
+```toml
+addr_book_strict = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+Set it to `false` for testing on private network. Most production nodes can keep it at `true`.
+
+### p2p.max_num_inbound_peers
```

Comment:
Maybe is missing in this section the explanation of this parameter itself: it is the maximum number of connections accepted by a node, while been started by peers. This should be seen as in a "slot" mode. Once an inbound connection is dropped, a new one can be added in "its place".